lib/catalog/rbac/access.rb

```diff
@@ -40,6 +40,13 @@ def resource_check(verb, id = @record.id, klass = @record.class)
         end
       end
 
+      def admin_access_check(table_name, verb)
```

Comment:
@eclarizio There is already a function defined in the common gem that you can call
https://github.com/RedHatInsights/insights-api-common-rails/blob/9b27c8413abf5a5e513ab2424f0f6c53c28c12ef/lib/insights/api/common/rbac/access.rb#L36

Reply:
Ah nice, forgot about that. Replaced.